src/pyflex/window_selector.py

```diff
@@ -688,10 +765,7 @@ def curtail_window_length(win):
             time_decay_right = self.config.min_period * self.config.c_4b / dt
             # Find all internal maxima.
             internal_maxima = self.peaks[
-                (self.peaks >= win.left) & (self.peaks <= win.right) &
-                (self.peaks != win.center)]
-            if len(internal_maxima) < 2:
-                return win
+                (self.peaks >= win.left) & (self.peaks <= win.right) ]
```

Comment:
I first compared yours with the original fortran version and I found the difference. I feel like the original fortran version makes more sense.  The curtailing left is just to curtail from the first left maxim. Same for the right maxim. It doesn't matter if the first left maxim is same as the center or the right maxium is same as the center. If in one window there is only one maxim(which is the center), it is still OK to curtail that window if the left or the right are too long. If you checked the FLEXWIN manual, in the curtail window example,  the example window just contains one maxim and it is curtailed.
What do you think?